### ISSUE TITLE
Replace Setting with Additional Shortcut

### ISSUE THREAD

**Issue by garyemerson** [janraasch/tab-ahead#81]
First off, I'm loving the extension! Unobtrusive, clean looking, performant and extremely useful.

What do you think about having the fuzzy search current window or all windows actions be exposed thru different shortcuts instead of a single shortcut and a setting. So pretty much it'd work something like 

`alt+t` => search current window
`alt+shift+t` => search all windows

Some benefits I see with this:
* Can easily and quickly switch between behaviors based on shortcut
* Even more similar to sublime, analogous to Goto Symbol and Goto Symbol in Project
* Dropping a setting makes the extension even simpler than it is now
* Can still simulate current behavior by only registering one shortcut for the action desired

**Comment by janraasch** [comment]
Hi @garyemerson, thank you for the kind words. I am glad you found this extension to be useful.

As to your suggested feature: In its current form this extension is a so called `chrome.browserAction` with a popup. As such we can specify *one* shortcut which opens the popup. You can find the documentation over at https://developer.chrome.com/extensions/browserAction. (This is way I created the settings page when adding the functionality to choose between searching all windows or the current window.)

So in short: I can absolutely see where you coming from, but Chrome's API for a `chrome.browserAction` simply does not offer the glue for functionality needed for this.

I hope this makes sense to you. Let me know, if you have any further questions.

**Comment by garyemerson** [comment]
Ah I see, that's a shame. Thanks for the consideration and quick response!
